ledger/sandbox-classic/.gitattributes

```diff
@@ -0,0 +1,3 @@
+# Enforce Unix newlines
```

Comment:
Why? Is this going to affect the Windows build?

Reply:
Good point, this file is currently present in `//ledger/sandbox` and got copied to the new `sandbox-classic` when I split it. But all SQL files moved to `//ledger/participant-integration-api`, so this file should be deleted.